docs/plugins/common.mdx

```diff
@@ -0,0 +1,709 @@
+| `getLastSelectedGuildId`   |                                | string \| undefined                     | Gets the last selected guild id                                       |
+| `getLastSelectedTimestamp` | <Required /> `guildId`: string | number                                  | Gets the timestamp of when a specific guild was last selected         |
+| `getState`                 |                                | `State`                                 | Gets an object containing some information about guild selections     |
+| `getTabsV2SelectedGuildId` |                                | string \| undefined                     | Gets the id of the last selected guild or the first guild in the list |
+| `isLoaded`                 |                                | boolean                                 | Gets a value indicating whether all initial guilds have been loaded   |
+
+`State` Props:
+
+| Name                           | Type                   | Description                                               |
+| ------------------------------ | ---------------------- | --------------------------------------------------------- |
+| `lastSelectedGuildId`          | string                 | Last selected guild id                                    |
+| `selectedGuildId`              | string                 | Current selected guild id                                 |
+| `selectedGuildTimestampMillis` | Record<string, number> | Object containing timestamps of when guilds were selected |
+
+### `messages`
+
+<!-- [!]: `CaptchaPayload`, `FetchMessagesOptions`, `MessageJumpOptions`, `MessagesData` and all functions starting with an underscore aren't documented -->
+
+All functions for message actions are in this module.
+
+Functions:
+
+| Name                            | Parameters                                                                                                                                                                                         | Return Type                                          | Description                                                  |
+| ------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- | ------------------------------------------------------------ |
+| `clearChannel`                  | <Required /> `channelId`: string                                                                                                                                                                   | void                                                 | Clears a channel from all messages                           |
```

Comment:
Change description: `Clears all messages in a channel. This will only apply locally and the messages will be re-fetched.`